src/type-extensions/error.ts

```diff
@@ -23,7 +23,7 @@ class ErrorTypeExtension extends TypeExtension<Error> {
             const metatableIndex = thread.lua.lua_gettop(thread.address)
 
             // Mark it as uneditable
-            thread.lua.lua_pushstring(thread.address, 'protected metatable')
+            thread.lua.lua_pushliteral(thread.address, 'protected metatable')
```

Comment:
I'm not certain using pushliteral instead of pushstring will make a difference. I think because all these strings are generated in JS rather than compiled into C code none of them count as literals.

Reply:
If I were doing a review I would probably say to put a comment somewhere explaining the preference of pushliteral over pushstring